modules/media/jsx/mediaIndex.js

```diff
@@ -12,7 +12,12 @@ class MediaIndex extends React.Component {
     this.state = {
       isLoaded: false,
       filter: {},
-      hiddenHeaders: ['Cand ID', 'Session ID', 'Hide File', 'File Type']
+      headers: {
```

Comment:
These don't seem to change anywhere, maybe they should be a class property or a const instead of state?

Reply:
The reason I'm setting them as a state is for the potential toggling of column visibility in the future. Based on our UX discussions it seems like column headers may become configurable and toggle-able in the future.

Reply:
I don't know if that's a good idea.. it seems like it would teach people to misuse state instead of understanding it.

Reply:
Hmm, okay. So i'll set both `headers.hidden` and `headers.all` to a const object?

Reply:
@driusan what do you think?